jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsHelper.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.nodejs
+
+import com.intellij.lang.javascript.buildTools.npm.PackageJsonUtil
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ProjectFileIndex
+import com.intellij.openapi.vfs.VirtualFile
+
+/**
+ * WebStorm doesn't allow user to mark a folder as source root. This method infers a folder as source root based on
+ * whether it has package.json file in it. If there is no package.json found in the path, content root will be returned.
+ *
+ * @param project The Node.js project.
```

Comment:
can we use kdoc instead of java doc?

Reply:
Isn't this KDoc? I found the kdoc link from here https://kotlinlang.org/docs/reference/kotlin-doc.html